### PR TITLE
Address is shown/instantiated for a newly approved member (company edit page)

### PR DESCRIPTION
**Pull request by weedySeaDragon**
PT Story:  New Member: editing their company -address fields not shown
https://www.pivotaltracker.com/story/show/148622121

When a newly approved member went to edit his/her company page, the address fields were not being shown because no address was instantiated. 

## Changes proposed in this pull request:
1.  Created a feature that goes through the entire process of a user applying for membership, an admin approving it, and the user (now a member) logging in and editing their company information.   (We didn't have a feature that went through the entire process, so this error was not being caught. )
1.  `kommun` is optional for an Address (since when the admin approves the application and the company is created, we need to have a `main_address,` but the kommun might not be known at that point.  [Otherwise we'll need to have a default kommun to assign to new, empty addresses.]
2.  Company.`main_address` will instantiate an Address when called
3.  Pull 'address' part of the company form view into a partial (first step toward being able to show multiple addresses

Ready for review:
@thesuss @RobertCram @patmbolger

### CODE REVIEW

**Review by patmbolger — APPROVED** [review]
Looks good.

File: app/controllers/companies_controller.rb [hunk 1]
```diff
@@ -55,6 +55,8 @@ def edit
 
     Ckeditor::Picture.images_category = 'company_' + @company.id.to_s
     Ckeditor::Picture.for_company_id  = @company.id
+
+    @company.main_address # this will create one if it's nil ( == lazy initialization)
   end
```
Comment (RobertCram) [inline app/controllers/companies_controller.rb]: Being lazy initialisation, I don't think we need this line here.

File: features/user-applies-approved-edits-co.feature [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    And I should see t("companies.show.company_number")
+    And I should see t("companies.telephone_number")
+    And I should see t("companies.show.email")
+    And I should see t("companies.show.street")
+    And I should see t("companies.show.post_code")
+    And I should see t("companies.show.city")
+    And I should see t("companies.show.kommun")
+    And I should see t("companies.show.region")
+    And I should see t("companies.show.website")
+
```
Comment (RobertCram) [inline features/user-applies-approved-edits-co.feature:67]: I think we should also add a feature for the sad path (where company name and the address fields are not filled in by the member). At the moment they are shown as required fields, but there are no error messages when left blank.

**Review by RobertCram — CHANGES_REQUESTED** [review]

File: features/user-applies-approved-edits-co.feature (hunk 2 above)
Comment (RobertCram) [reply features/user-applies-approved-edits-co.feature:67]: Sorry, I see that's supposed to be another story (https://www.pivotaltracker.com/story/show/141168775)

**Review by RobertCram — APPROVED** [review]

**Pull request merged**